akka-d3-readside-cassandra/src/main/resources/reference.conf

```diff
@@ -24,37 +24,47 @@ akka {
 }
 
 cassandra-readside {
-  # Comma-separated list of contact points in the Cassandra cluster
+  # FQCN of the cassandra journal plugin
```

Comment:
@pjan I copied the `cassandra-journal` section from `akka-persistence-cassandra`'s reference configuration in the `cassandra-readside` section so we can see easily what changed between versions. Let me know what to add/rewrite/take out.

(There are also `cassandra-snapshot-store` and `cassandra-query-journal` sections in the `akka-persistence-cassandra` reference configuration; I didn't add them here.)

https://github.com/akka/akka-persistence-cassandra/blob/master/core/src/main/resources/reference.conf#L15